HiP-ThumbnailService/Arguments/RequestedImageFormat.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+
+namespace PaderbornUniversity.SILab.Hip.ThumbnailService.Arguments
+{
+    public enum RequestedImageFormat
+    {
+        Jpeg, Png
+
+    }
+
+    public class RequestImageFormatUtils
```

Comment:
This class should be static. `GetExtension` could also be an extension method then.